internal/access/organization.go

```diff
@@ -44,7 +44,7 @@ func CreateOrganization(c *gin.Context, org *models.Organization) error {
 		return HandleAuthErr(err, "organizations", "create", models.InfraSupportAdminRole)
 	}
 
-	return data.CreateOrganizationAndSetContext(db, org)
+	return data.CreateOrganization(db, org)
```

Comment:
seems like the removal of CreateOrganizationAndSetContext would cause problems? How does the current context or tx get set to the new org? is this the responsibility of the caller now?

Reply:
It is the responsibility of the caller now.